### PR TITLE
MMCA-4996 Change FileRole value for cash account statement

### PR DESCRIPTION
**Pull request by harshare**
Due to ETMP using a different filerole value in cash statement metadata, FileRole value for cash statement is changed from 'CashStatement' to 'CDSCashAccount'

### CODE REVIEW

**Review by AdamCoops — DISMISSED** [review]

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -17,15 +17,7 @@
...
-import models.{
-  C79Certificate,
-  CashStatement,
-  DutyDefermentStatement,
-  FileRole,
-  PostponedVATStatement,
-  SecurityStatement,
-  UserAnswers
-}
+import models.{C79Certificate, CDSCashAccount, DutyDefermentStatement, FileRole, PostponedVATStatement, SecurityStatement, UserAnswers}
```
Comment (tyagikumarvineet) [inline app/config/FrontendAppConfig.scala:20]: Better to revert as the last style (imports on separate line) or break after PostponedVATStatement as it making a very long line

File: app/config/FrontendAppConfig.scala [hunk 2]
```diff
@@ -71,7 +63,7 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
   lazy val sdesDutyDefermentStatementListUrl: String = sdesApi + "/files-available/list/DutyDefermentStatement"
   lazy val sdesImportVatCertificateListUrl: String = sdesApi + "/files-available/list/C79Certificate"
   lazy val sdesImportPostponedVatStatementListUrl: String = sdesApi + "/files-available/list/PostponedVATStatement"
-  lazy val sdesCashStatementListUrl: String = s"$sdesApi/files-available/list/CashStatement"
+  lazy val sdesCashStatementListUrl: String = s"$sdesApi/files-available/list/CDSCashAccount"
```
Comment (tyagikumarvineet) [inline app/config/FrontendAppConfig.scala:66]: Could you please add test for this?

File: app/config/FrontendAppConfig.scala [hunk 3]
```diff
@@ -115,7 +107,7 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
       case C79Certificate => c79ReturnLink
       case SecurityStatement => adjustmentsReturnLink
       case PostponedVATStatement => postponedVatReturnLink
-      case CashStatement => cashStatementReturnLink
+      case CDSCashAccount => cashStatementReturnLink
```
Comment (tyagikumarvineet) [inline app/config/FrontendAppConfig.scala:110]: Could you please add a test case for this change?

File: app/models/FileRole.scala [hunk 4]
```diff
@@ -49,7 +49,7 @@ object FileRole {
         case "duty-deferment" => Right(DutyDefermentStatement)
         case "adjustments" => Right(SecurityStatement)
         case "postponed-vat" => Right(PostponedVATStatement)
-        case "cash-statement" => Right(CashStatement)
+        case "cash-statement" => Right(CDSCashAccount)
```
Comment (tyagikumarvineet) [inline app/models/FileRole.scala:52]: Could you please add test case for this and change at line 63 as well? If it takes time then leave it

File: test/config/FrontendAppConfigSpec.scala [hunk 5]
```diff
@@ -43,7 +43,7 @@ class FrontendAppConfigSpec extends SpecBase {
     }
 
     "return the adjustments link if a CashStatement FileRole provided" in new Setup {
```
Comment (tyagikumarvineet) [inline test/config/FrontendAppConfigSpec.scala:45]: Could you please replace CashStatement with CDSCashAccount?

File: test/connectors/SdesConnectorSpec.scala
Comment (tyagikumarvineet) [inline test/connectors/SdesConnectorSpec.scala:1]: Could you please  replace CashStatement with CDSCashAccount at line 172 and 178?

File: test/models/FileRoleSpec.scala [hunk 6]
```diff
@@ -39,7 +39,7 @@ class FileRoleSpec extends SpecBase {
     }
 
     "return a JsSuccess for CashStatement" in {
```
Comment (tyagikumarvineet) [inline test/models/FileRoleSpec.scala:41]: Could you please replace CashStatement with CDSCashAccount?

File: app/config/FrontendAppConfig.scala (hunk 1 above)
Comment (harshare) [reply app/config/FrontendAppConfig.scala:20]: Reverted to previous style

File: app/config/FrontendAppConfig.scala (hunk 2 above)
Comment (harshare) [reply app/config/FrontendAppConfig.scala:66]: Added test for sdesCashStatementListUrl

File: app/config/FrontendAppConfig.scala (hunk 3 above)
Comment (harshare) [reply app/config/FrontendAppConfig.scala:110]: Added tests for this change

File: app/models/FileRole.scala (hunk 4 above)
Comment (harshare) [reply app/models/FileRole.scala:52]: Added tests

File: test/config/FrontendAppConfigSpec.scala (hunk 5 above)
Comment (harshare) [reply test/config/FrontendAppConfigSpec.scala:45]: Replaced CashStatement with CDSCashAccount

File: test/models/FileRoleSpec.scala (hunk 6 above)
Comment (harshare) [reply test/models/FileRoleSpec.scala:41]: Replaced CashStatement with CDSCashAccount

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**